src/ray/common/test_util.cc

```diff
@@ -45,6 +45,14 @@ void RedisServiceManagerForTest::TearDownTestCase() {
   usleep(100 * 1000);
 }
 
+void RedisServiceManagerForTest::FlushAll() {
+  std::string flush_all_redis_command =
+      REDIS_CLIENT_EXEC_PATH + " -p " + std::to_string(REDIS_SERVER_PORT) + " flushall";
+  RAY_LOG(INFO) << "Clear redis command is: " << flush_all_redis_command;
```

Comment:
`Cleaning up redis with command: ...`
And should this be a debug log?

Reply:
I think the `INFO` level is better  for `FlushAll`, it is the same as `SetUpTestCase` and `TearDownTestCase`.